api/src/templates/certificate/index.js

```diff
@@ -86,6 +88,11 @@ const phase1 = async (young, batch = false) => {
   const departureDate = await getDepartureDateSession(meetingPoint, session, young, cohort);
   const returnDate = await getReturnDateSession(meetingPoint, session, young, cohort);
 
+  if (!cohesionCenter) {
+    capture(ERRORS.NO_COHESION_CENTER_FOUND);
+    throw ERRORS.NO_COHESION_CENTER_FOUND;
```

Comment:
```suggestion
    throw { message: ERRORS.NO_COHESION_CENTER_FOUND };
```
pour rentrer dans le `captureMessage`